SetLibrary/SetLibrary/SetLibrary/NumSet.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+
+namespace SetLibrary
+{
+    using List = System.Collections.Generic.LinkedList<int>;
+
+    public class NumSet
+    {
+        List Elements;
```

Comment:
```suggestion
        private readonly HashSet<int> _Elements;
```
Preferably, be explicit in your access declarations. Also, this should probably be a HashSet, not a LinkedList.